src/injected/element-finder-by-position.ts

```diff
@@ -46,7 +45,7 @@ export class ElementFinderByPosition {
             return { payload: path };
         }
 
-        path.push(this.scannerUtils.getUniqueSelector(element));
+        path.push(await this.getUniqueSelector(element));
```

Comment:
Do we need the `await` on this line? `getUniqueSelector` is not declared `async` above.

Reply:
Nope! This was leftover from when I was experimenting with having it wait for any ongoing scans to complete, removed